app/upgrades/types.go

```diff
@@ -1,13 +1,20 @@
 package upgrades
 
 import (
+	"time"
+
 	storetypes "cosmossdk.io/store/types"
 	upgradetypes "cosmossdk.io/x/upgrade/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/x/authz"
 
 	"github.com/pokt-network/poktroll/app/keepers"
 )
 
+var testNetPnfAddress = "pokt1r6ja6rz6rpae58njfrsgs5n5sp3r36r2q9j04h"
+var testNetAuthorityAddress = "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t"
```

Comment:
```suggestion
var (
	// The default PNF/DAO address in the genesis file for local development
	testNetPnfAddress = "pokt1r6ja6rz6rpae58njfrsgs5n5sp3r36r2q9j04h"
	
	// A new authority address to which the PNF/DAO address above authorizes to act on its behalf
	testNetAuthorityAddress = "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t"
)
```

But, I looked in the `Makefile` and found this:

```
GOV_ADDRESS = pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t
# PNF acts on behalf of the DAO and who AUTHZ must delegate to
PNF_ADDRESS = pokt1eeeksh2tvkh7wzmfrljnhw4wrhs55lcuvmekkw
```

Can you add comments so it's clear what/why we're doing here? I'm pretty confused.

Reply:
Moved vars and extended comments so it is obvious what it is and why.